chore(origin-ui) Next major version

arkadiusz-s opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/energywebfoundation/origin?pullRequest=2266) <br/>All committers have signed the CLA.

alexworker23 committed:
chore(origin-ui-localization): add missing localization

alexworker23 committed:
chore: improve mobile-view styling

alexworker23 committed:
chore(origin-ui-device-data): package clean up

alexworker23 committed:
feat(origin-ui-core): add back button for multi-step form

alexworker23 committed:
chore(origin-ui-organization-logic): adjust multi step forms to changes

alexworker23 committed:
fix(origin-ui-device): fix device register form

alexworker23 committed:
chore(origin-ui-localization): add missing localizations

alexworker23 committed:
chore(origin-ui-device-logic): format energy in pending table

alexworker23 committed:
chore(origin-ui-localization): add localization for pending devices

alexworker23 committed:
chore(origin-device-registry-api): adjust error text

alexworker23 committed:
fix(origin-ui-device-logic): make smartMeterId field required following api requirements

alexworker23 committed:
chore(origin-ui-core): fallback for autocomplete value to always keep component controlled

alexworker23 committed:
chore: deepscan fixes

driamend committed:
Merge pull request #2717 from energywebfoundation/chore/fixes-and-improvements

Chore/fixes and improvements

alexworker23 committed:
Merge branch 'chore/origin-ui-v2' into chore/ui-fixes-2

alexworker23 committed:
Merge pull request #2715 from energywebfoundation/chore/ui-fixes-2

chore/UI fixes

alexworker23 committed:
feat(origin-ui-core): add validationMode props for GenericForm, adjust button disabling

alexworker23 committed:
chore(origin-ui-user): adjust user login form styling and validation

alexworker23 committed:
chore(origin-ui-utils): packages clean up

alexworker23 committed:
fix(origin-ui-organization): use proper import for irecAccountType

alexworker23 committed:
chore(origin-ui-core): extend core components props

alexworker23 committed:
fix(origin-ui-device): stable displaying fallback icons in my devices

alexworker23 committed:
feat(origin-ui-core): pass back button as prop to custom parts of multistep form

alexworker23 committed:
chore(origin-ui-organization): adjust organization forms

alexworker23 committed:
chore(origin-ui-device): pass back button to multistep form

alexworker23 committed:
chore(origin-ui-user-view): adjust modal redirect path

alexworker23 committed:
chore(origin-ui-user-data): better organized file structure

alexworker23 committed:
chore(origin-ui-user): improve validation

alexworker23 committed:
chore(origin-ui-device): show text info when user has no devices

alexworker23 committed:
feat(origin-ui-localization): localization for noDevicesOwned block

driamend committed:
chore(origin-ui-core): improve datepicker style, add clear button, open on input click

driamend committed:
chore(origin-ui-core): improve datepicker style, add clear button, open on input click

alexworker23 committed:
feat(origin-ui-certificate): add validation for inbox actions

alexworker23 committed:
chore(origin-ui-core): adjust form components props

alexworker23 committed:
chore(origin-ui-localization): add missing localizations

josipbagaric committed:
chore: Don't log full order

alexworker23 committed:
chore(origin-ui): add memo for App

alexworker23 committed:
fix(origin-ui-exchange): multiple requests to orderbook on initial render

alexworker23 committed:
feat(origin-ui-exchange): display multiple device types for bid

alexworker23 committed:
chore(origin-ui-utils): adjust type

alexworker23 committed:
chore(origin-ui-certificate-logic): remove unused imports

alexworker23 committed:
feat(origin-ui-exchange): create bundle action validation

alexworker23 committed:
chore: packages clean up

alexworker23 committed:
Merge pull request #2718 from energywebfoundation/fix/ui-fixes-3

fix/ui fixes and improvements

driamend committed:
chore(origin-ui-localization): add general requirements

driamend committed:
chore(origin-ui): add usePermissions hook

driamend committed:
chore(origin-ui-core): add Requirements and PermissionsFeedback components

driamend committed:
chore(origin-ui-exchange): add Requirements to exchange pages

driamend committed:
chore(origin-ui-device): add Requirements to device pages

driamend committed:
chore(origin-ui-certificate): add Requirements to certificate pages

driamend committed:
Merge branch 'chore/fixes-and-improvements-2' of github.com:energywebfoundation/origin into chore/fixes-and-improvements-2

alexworker23 committed:
chore: prepare react-query api clients for publishing

alexworker23 committed:
chore(origin-ui-user-data): remove exports

alexworker23 committed:
Merge branch 'master' into chore/prep-for-publish

alexworker23 committed:
chore: align deps versions

driamend committed:
Merge branch 'chore/origin-ui-v2' of github.com:energywebfoundation/origin into chore/fixes-and-improvements-2

alexworker23 committed:
chore(origin-ui-core): remove localization of notification type from core component

alexworker23 committed:
chore: remove unused package

alexworker23 committed:
chore: move initializeI18 to localization package from utils

alexworker23 committed:
chore: remove cross-package dependencies

alexworker23 committed:
WIP: build all ui-packages separately

alexworker23 committed:
chore: update nx to version 12

driamend committed:
chore(origin-ui-core): move PermissionsFeedback to Requirements, pass title as prop

driamend committed:
chore(origin-ui): remove api calls from usePermissions

driamend committed:
chore(origin-ui-user): remove exports

driamend committed:
chore(origin-ui-exchange): add and use useApiPermissions

driamend committed:
chore(origin-ui-device): add and use useApiPermissions

driamend committed:
chore(origin-ui-certificate): add and use useApiPermissions

alexworker23 committed:
chore: add custom rollup config

alexworker23 committed:
chore(origin-ui-core): fix build

alexworker23 committed:
chore: remove circullar dependencies

alexworker23 committed:
chore: add license, readme's update package.json's

alexworker23 committed:
chore: build all core packages, remove babelConfig prop

driamend committed:
chore(origin-ui): remove permissions from utils

driamend committed:
chore(origin-ui-core): change Requirements props

driamend committed:
chore(origin-ui-certificate): add userAndAccount and usePermissionsLogic

driamend committed:
chore(origin-ui-certificate): add permissions to pages

driamend committed:
chore(origin-ui-device): add userAndAccount and usePermissionsLogic

driamend committed:
chore(origin-ui-device): add permissions to pages

driamend committed:
chore(origin-ui-exchange): add userAndAccount and usePermissionsLogic

driamend committed:
chore(origin-ui-exchange): add permissions to pages

driamend committed:
chore(origin-ui-certificate): remove t function from useBlockchainInboxPermissionsLogic

driamend committed:
chore(origin-ui): remove permissions and requirement types

driamend committed:
chore(origin-ui-certificate): fix import

driamend committed:
chore(origin-ui-core): fix missing type

driamend committed:
Merge pull request #2722 from energywebfoundation/chore/fixes-and-improvements-2

Chore/fixes and improvements 2

alexworker23 committed:
chore: configure build for all ui packages

alexworker23 committed:
Merge branch 'chore/origin-ui-v2' into chore/prep-for-publish

alexworker23 committed:
chore(origin-ui-exchange): remove circular dependencies inside the packages

alexworker23 committed:
chore: build packages-ui with deps instead of peerDeps

alexworker23 committed:
chore: extend rollup config with used globals

alexworker23 committed:
chore: lint fix

alexworker23 committed:
Merge pull request #2723 from energywebfoundation/chore/prep-for-publish

chore/prepare packages for publishing

alexworker23 committed:
feat(origin-ui-core): make error fallback more customizable

alexworker23 committed:
feat(origin-ui): add styled error fallback

alexworker23 committed:
chore(origin-ui-localization): add missing localization for errors

alexworker23 committed:
chore(origin-ui-view): remove requirements from view-market page

alexworker23 committed:
fix(origin-ui-user): mmemory leak on fetching exchange address

alexworker23 committed:
chore(origin-ui-certificate-view): remove requirements from pending-certificates page

alexworker23 committed:
chore(origin-ui-organization): remove owner name from roleChangedModal

alexworker23 committed:
chore(origin-ui-user): remove owner name from roleChangedModal

alexworker23 committed:
chore(origin-ui-localization): remove owner name from roleChangedModal

alexworker23 committed:
Merge branch 'master' into chore/ui-improvements

alexworker23 committed:
chore: adjust to merged changes

alexworker23 committed:
feat(origin-ui-certificate): add approved page for issuer

alexworker23 committed:
chore(origin-ui): add approved certificates page

alexworker23 committed:
feat(origin-ui-localization): add approved certificates page localization

alexworker23 committed:
Merge pull request #2737 from energywebfoundation/chore/ui-improvements

chore/UI improvements

alexworker23 committed:
chore(origin-ui-shared-state): remove unused

driamend committed:
chore(origin-ui-exchange): render endAdornment on energy and price inputs

driamend committed:
chore(origin-ui-core): fix table overflow

driamend committed:
chore(origin-ui-exchange): improve view market styles

driamend committed:
Merge pull request #2739 from energywebfoundation/chore/fixes-and-improvements-3

Chore/fixes and improvements 3

alexworker23 committed:
chore: prepare ui for deployment, split env variables

alexworker23 committed:
chore: update env example

alexworker23 committed:
chore(origin-ui): adjust nginx port

alexworker23 committed:
Merge pull request #2738 from energywebfoundation/feat/prep-ui-deploy

feat/prep UI deploy

driamend committed:
chore(origin-ui-exchange): reset dates on create bid, add dates validation on create demand, disable submit button on submitting

driamend committed:
Merge pull request #2741 from energywebfoundation/chore/view-market-fixes

chore(origin-ui-exchange): reset dates on create bid, add dates valid…

alexworker23 committed:
chore: fix build

alexworker23 committed:
chore: add dseparate deployment of new ui

alexworker23 committed:
Merge pull request #2742 from energywebfoundation/chore/fix-build

chore/fix build

alexworker23 committed:
Merge branch 'master' into chore/origin-ui-v2

alexworker23 committed:
fix(origin-ui-exchange): fetching user for exchange forbiden when unlogged

driamend committed:
chore(origin-ui-certificate): fix map marker position

driamend committed:
chore(origin-ui-device): handle import device error

driamend committed:
Merge pull request #2744 from energywebfoundation/chore/ui-fixes

Chore/UI fixes

alexworker23 committed:
feat(origin-ui-core): add TimeframeSelect

alexworker23 committed:
chore: adjust storybook run command

alexworker23 committed:
feat(origin-ui-core): add CardsList component

alexworker23 committed:
feat(origin-ui-core): add CardsTable component

alexworker23 committed:
Merge pull request #2745 from energywebfoundation/feat/add-new-core-components

feat/add new core components

alexworker23 committed:
chore: adjust to strict ts rules

alexworker23 committed:
chore(origin-ui-device-data): forbid requesting certificates for same time period"

alexworker23 committed:
chore(origin-ui-localization): add notification localization

alexworker23 committed:
Merge pull request #2748 from energywebfoundation/chore/strict-ts-rules

chore/strict ts rules

josipbagaric committed:
Merge branch 'master' into chore/origin-ui-v2

josipbagaric committed:
chore(ui): fix build

josipbagaric committed:
Merge pull request #2769 from energywebfoundation/fix/build

chore(ui): fix build

josipbagaric committed:
chore: fix version mismatch

alexworker23 committed:
chore: improve storybook config, update versions

alexworker23 committed:
docs(origin-ui-core): added storybook docs for buttons components

alexworker23 committed:
docs(origin-ui-core): add storybook docs for form components

alexworker23 committed:
feat(origin-ui-core): added loading state for form containers

alexworker23 committed:
chore(origin-ui-theme): added export of ui-config creation func

alexworker23 committed:
chore(origin-ui-exchange): adjust to changes in form props

alexworker23 committed:
chore(origin-ui-device): added loading state for request-cert and register-device forms

alexworker23 committed:
chore(origin-ui): minor adjustments, added error fallback button actions

alexworker23 committed:
Merge branch 'chore/origin-ui-v2' into chore/form-storybook

alexworker23 committed:
chore(origin-ui-exchange): remove unused

josipbagaric committed:
chore(ui): Show error message to the user

alexworker23 committed:
chore: update lockfile

josipbagaric committed:
Merge pull request #2770 from energywebfoundation/chore/error-msg

chore: Exact error message

josipbagaric committed:
Merge branch 'master' into chore/origin-ui-v2

alexworker23 committed:
Merge pull request #2765 from energywebfoundation/chore/form-storybook

chore/form storybook

alexworker23 committed:
chore: extended error messages

alexworker23 committed:
feat(origin-ui-core): add prop to allow submitting form with initialValues

alexworker23 committed:
fix(origin-ui-exchange-logic): fix buy direct form validation

alexworker23 committed:
chore(origin-ui-device-logic): adjust request certificate form validation text

alexworker23 committed:
chore(origin-ui-localization): add missing localization for myTrades

alexworker23 committed:
fix(origin-ui-certificate): provide pending status for blockchain transaction in inbox

alexworker23 committed:
feat(origin-ui-core): optional disabled state for all list components

alexworker23 committed:
fix(origin-ui-core): show partiallyfilled asks and bids in my-orders

alexworker23 committed:
chore(origin-ui-localization): add missing localization)

alexworker23 committed:
fix: display success icon in error notifications

alexworker23 committed:
chore: add null check and remove unused import

alexworker23 committed:
Merge pull request #2772 from energywebfoundation/fix/bugfixes

fix/ui bug fixes

alexworker23 committed:
chore: hide routes for user following the menu items display

alexworker23 committed:
chore(origin-ui-device-view): make image upload optional in device registration

alexworker23 committed:
chore(origin-ui-device-view): make image upload optional in device registration

alexworker23 committed:
chore(origin-ui): adjust export

alexworker23 committed:
feat(origin-ui-certificate-logic): clicking on row in claims report leads to certificate detail view

alexworker23 committed:
feat(origin-ui-certificate): add loading state for exchange inbox

alexworker23 committed:
chore(origin-ui-core): notifications get font family from mui-theme

alexworker23 committed:
Merge pull request #2789 from energywebfoundation/fix/device-images

fix/device images

alexworker23 committed:
Merge pull request #2788 from energywebfoundation/fix/routing-ui

fix/routing

alexworker23 committed:
feat(origin-ui-certificate): extend certificate detail view with claim data

alexworker23 committed:
feat(origin-ui-localization): add claim data localization

alexworker23 committed:
feat(origin-ui-certificate-view): add view device button for certificate detail view

alexworker23 committed:
chore(origin-ui-localization): add localization for view device button

alexworker23 committed:
fix(origin-ui-exchange-data): filtering asks and bids by status

alexworker23 committed:
Merge pull request #2790 from energywebfoundation/feat/cert-claim-data

feat/cert claim data

alexworker23 committed:
chore: add missing types for args

alexworker23 committed:
Merge branch 'master' into chore/alpha_release_ui-v2

alexworker23 committed:
chore: align packages versions

alexworker23 committed:
chore(origin-ui-certificate): filtering and displaying events in detail view

alexworker23 committed:
chore: update rollup config and utils version

alexworker23 committed:
chore(origin-ui-core): allow custom icons for navigation components

alexworker23 committed:
chore(origin-ui-theme): remove themeProvider

alexworker23 committed:
chore(origin-ui): add theme provider

alexworker23 committed:
chore(ui): adjust build and deps for core packages

alexworker23 committed:
Merge pull request #2792 from energywebfoundation/chore/alpha_release_ui-v2

chore/alpha release UI v2

alexworker23 committed:
feat(origin-ui-core): allow custom icon for mainLayout

alexworker23 committed:
Merge pull request #2808 from energywebfoundation/feat/custom-icon-sidebar

feat/custom icon for mainlayout

alexworker23 committed:
chore(origin-ui-certificate): use context for env vars instead of extracting from process

alexworker23 committed:
chore(origin-ui-device): use context for env vars instead of extracting from process

alexworker23 committed:
chore(origin-ui-user): use context for env vars instead of extracting from process

alexworker23 committed:
chore(origin-ui): pass env variables from process.env to ui apps

alexworker23 committed:
chore: remove unused imports

alexworker23 committed:
chore(origin-ui-core): minor improvements to GenericForm types and defaults

alexworker23 committed:
docs(origin-ui-core): add storybook docs for GenericForm

alexworker23 committed:
chore(origin-ui-core): improve typings in MultiStepForm

alexworker23 committed:
docs(origin-ui-core): storybook docs for MultiStepForm

alexworker23 committed:
fix(origin-ui-user-data): wrong return type in submitHandlers

alexworker23 committed:
fix(origin-ui-device-view): submitHandler type

alexworker23 committed:
fix(origin-ui-organization-view): submitHandler type and loading state for org-register form

alexworker23 committed:
Merge pull request #2809 from energywebfoundation/chore/improve_ui_env_handling

chore/improve ui env handling

alexworker23 committed:
Merge pull request #2814 from energywebfoundation/feat/form-containers-docs

feat/form containers docs

alexworker23 committed:
chore(origin-ui-device): disconnect meter reads block from mocks

alexworker23 committed:
chore: remove unused deps

alexworker23 committed:
feat(origin-ui-core): extend CardWithImage and related components props to allow more customization

alexworker23 committed:
Merge pull request #2816 from energywebfoundation/chore/remove-mocks

chore/remove mocks

alexworker23 committed:
refactor(origin-ui-core): remove notifications-center component

alexworker23 committed:
refactor(origin-ui): create NotificationsCenter in root instead of importing

alexworker23 committed:
feat(origin-ui-user-view): add customizable bgImage and icon for login page

alexworker23 committed:
refactor(origin-ui-assets): color selected icons via css

alexworker23 committed:
chore(origin-ui-theme): remove default exports

alexworker23 committed:
Merge pull request #2818 from energywebfoundation/chore/ui_improvements

chore/ui-v2 improvements

alexworker23 committed:
chore(origin-ui-theme): clean up theme config

alexworker23 committed:
Merge pull request #2819 from energywebfoundation/chore/clean_theme-config

chore/clean theme config

alexworker23 committed:
fix(origin-ui-device): display uploaded device images

alexworker23 committed:
Merge pull request #2821 from energywebfoundation/fix/device_images

fix/device images

alexworker23 committed:
refactor(origin-ui-organization-data): improve folder structure, separate fetching and handlers functions

alexworker23 committed:
Merge branch 'master' into feat/connect-irec-ui

alexworker23 committed:
chore: update deps

alexworker23 committed:
feat(origin-ui-core): add disabled prop for GenericForm

alexworker23 committed:
feat(origin-ui-assets): add i-rec logo

alexworker23 committed:
feat(origin-ui-organization): add connect-irec page

alexworker23 committed:
feat(origin-ui): add connect-irec page

alexworker23 committed:
feat(origin-ui-localization): add localization for connect-irec page

alexworker23 committed:
feat(origin-ui-device-view): add fallback notification for empty all-devices page

alexworker23 committed:
chore(origin-ui-core): adjusted default styling for CardWithImage and related components

alexworker23 committed:
refactor(origin-ui-core): make imageUrl required prop for CardWithImage component, connect new default styling

alexworker23 committed:
docs(origin-ui-core): storybook docs for CardWithImage component

alexworker23 committed:
chore(origin-ui-core): adjust HorizontalCard props and types

alexworker23 committed:
docs(origin-ui-core): storybook docs for HorizontalCard component

alexworker23 committed:
Merge pull request #2822 from energywebfoundation/feat/connect-irec-ui

feat/irec connection ui

alexworker23 committed:
Merge pull request #2838 from energywebfoundation/docs/cards-components

docs/cards components

alexworker23 committed:
feat(origin-ui-core): prepare package for double theming
add ThemeSwitcher component
adjust core components theme supplying for better view in light theme
extend nav and layout component to allow more customization

alexworker23 committed:
feat(origin-ui-shared-state): add themeMode localStorage management functions

alexworker23 committed:
feat(origin-ui-theme): add light theme color scheme, extend config for double-theming

alexworker23 committed:
feat(origin-ui-assets): add dark I-REC logo

alexworker23 committed:
feat(origin-ui): add light theme config, custom styles for light theme, improve folder structure

alexworker23 committed:
feat: adjust views and menus configuration to work with light theme

alexworker23 committed:
refactor: use ThemeModeEnum instead string

alexworker23 committed:
Merge pull request #2841 from energywebfoundation/feat/theme-switcher

feat/theme switcher

alexworker23 committed:
Merge branch 'master' into chore/origin-ui-v2

alexworker23 committed:
chore: update deps

alexworker23 committed:
chore: adjust to merged changes

alexworker23 committed:
fix(origin-organization-irec-api): apiBody type
